torchtune/modules/model_fusion/_deep_fusion.py

```diff
@@ -97,15 +94,10 @@ def __init__(
             trainable_params -= set(get_fusion_params(self))
         set_trainable_params(self, trainable_params)
 
+    @deprecated("Please use self.skip_output=True and use a linear loss instead")
```

Comment:
nit but I feel like "use a linear loss" is not really sufficient to describe how someone should migrate off of this